src/test/java/com/commercetools/sync/types/utils/TypeUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,104 @@
+package com.commercetools.sync.types.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.types.FieldDefinition;
+import io.sphere.sdk.types.ResourceTypeIdsSetBuilder;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.TypeDraft;
+import io.sphere.sdk.types.TypeDraftBuilder;
+import io.sphere.sdk.types.commands.updateactions.ChangeName;
+import io.sphere.sdk.types.commands.updateactions.SetDescription;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.commercetools.sync.types.FieldDefinitionTestHelper.imageUrlFieldDefinition;
+import static com.commercetools.sync.types.FieldDefinitionTestHelper.relatedCategoriesFieldDefinition;
+import static com.commercetools.sync.types.FieldDefinitionTestHelper.stateFieldDefinition;
+import static com.commercetools.sync.types.utils.TypeUpdateActionUtils.buildChangeNameUpdateAction;
+import static com.commercetools.sync.types.utils.TypeUpdateActionUtils.buildSetDescriptionUpdateAction;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class TypeUpdateActionUtilsTest {
+    private static Type old;
+    private static TypeDraft newSame;
+    private static TypeDraft newDifferent;
+
+    /**
+     * Initialises test data.
+     */
+    @BeforeClass
+    public static void setup() {
+        final String key = "category-custom-type-key";
+        final LocalizedString name = LocalizedString.ofEnglish("type for standard categories");
+        final LocalizedString desc = LocalizedString.ofEnglish("description for category custom type");
+
+        //this enables the type only to be used for categories
```

Comment:
I think this is explained in the docs of CTP, so it's not very needed here.

Reply:
same : https://github.com/commercetools/commercetools-sync-java/pull/333#discussion_r236313090